examples/can/python-can/kvaser/send_and_receive_packet.py

```
@@ -61,8 +61,10 @@ def main():
     print(received_packet_record)
 
     # close connections with CAN interfaces
-    del can_ti_1
-    del can_ti_2
+    can_ti_1.teardown_sync(True)
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)